var/spack/repos/builtin.mock/packages/flake8/package.py

```diff
@@ -30,7 +30,7 @@ class Flake8(Package):
     All of these violations are exceptions that we allow in
     package.py files."""
 
-    # Used to tell whether or not the package has been modified
+    # Used to tell whether or not the package has been unmodified
```

Comment:
I think modified sounds better here. I had the same thing too but I ended up changing it.